rasa/core/test.py

```diff
@@ -802,11 +860,21 @@ async def test(
             report_filename = os.path.join(out_directory, REPORT_STORIES_FILE)
             rasa.shared.utils.io.dump_obj_as_json_to_file(report_filename, report)
             logger.info(f"Stories report saved to {report_filename}.")
+
         else:
             report, precision, f1, accuracy = get_evaluation_metrics(
                 targets, predictions, output_dict=True
             )
 
+        evaluate_entities(
```

Comment:
don't we want to show the failed entity predictions within the stories? This would dump them in a separate file, right? My intuition would have been to merge our policy predictions with the NLU predictions which were made in `_collect_user_uttered_predictions`

Reply:
As discussed with @wochinge, we will keep the scope to evaluating entities in separate files like for the NLU components as there is a bug: https://github.com/RasaHQ/rasa/issues/8152 currently for displaying incorrect entity extracting in failed test stories.